src/openzeppelin/utils/constants/library.cairo

```diff
@@ -34,3 +34,13 @@ const IACCESSCONTROL_ID = 0x7965db0b;
 //
 
 const DEFAULT_ADMIN_ROLE = 0;
+
+//
+// Starknet
+//
+
+const TRANSACTION_VERSION = 1;
+// The offset for query transaction versions
+const QUERY_VERSION_BASE = 2**128;
+// For transactions meant to query and not to be executed by the Starknet OS
```

Comment:
```suggestion
// For transactions meant to simulate and not to be executed by the Starknet OS
```
I know "query" is the official word choice, but I find simulation/fee estimation more clear terms